<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_Login</class>
 <widget class="QDialog" name="Dlg_Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="PictureLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>691</width>
     <height>621</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../qrc.qrc">:/new/prefix1/qrcpicture/picture1.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="LibraryLabel">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>60</y>
     <width>301</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文仿宋</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">LibraryLabel {
    color: black;
}</string>
   </property>
   <property name="text">
    <string>图书借阅管理系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>367</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>登          录</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PasswordLayout">
    <item>
     <widget class="QLabel" name="PasswordLabel">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PasswordEdit">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>731</x>
     <y>436</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="RegisterLayout">
    <item>
     <widget class="QLabel" name="RegisterLabel">
      <property name="text">
       <string>        没有账号？   请联系管理员</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>170</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="UserLayout">
    <item>
     <widget class="QLabel" name="UserLabel">
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>10</pointsize>
        <fontweight>Medium</fontweight>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>用户：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UserEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="DraggableLabel" name="Draglabel">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>1</y>
     <width>851</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ToMinButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(0, 0, 0, 0); /* 完全透明 */
        border: none; /* 移除边框 */
        padding: 0px; /* 移除内边距 */
        width: 36px; /* 固定宽度 */
        height: 36px; /* 固定高度 */
        icon-size: 24px; /* 图标大小 */
    }
    
    QPushButton::icon {
        color:rgb(64, 64, 64); /* 深灰色图标 */
    }
    
    QPushButton:hover {
        background-color: rgba(0, 0, 0, 0.1); /* 轻微半透明灰色 */
    }
    
    QPushButton:pressed {
        background-color: rgba(0, 0, 0, 0.2); /* 点击时稍深半透明灰色 */
        border-radius: 2px; /* 圆角效果 */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListRemove"/>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseButton">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>0</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(0, 0, 0, 0); /* 完全透明 */
        border: none; /* 移除边框 */
        padding: 0px; /* 移除内边距 */
        width: 36px; /* 固定宽度 */
        height: 36px; /* 固定高度 */
        icon-size: 24px; /* 图标大小 */
    }
    
    QPushButton::icon {
        color: #333; /* 深灰色图标 */
    }
    
    QPushButton:hover {
        background-color: rgba(0, 0, 0, 0.1); /* 轻微半透明灰色 */
    }
    
    QPushButton:pressed {
        background-color: rgba(0, 0, 0, 0.2); /* 点击时稍深半透明灰色 */
        border-radius: 2px; /* 圆角效果 */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::WindowClose"/>
   </property>
   <property name="iconSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
  </widget>
  <zorder>PictureLabel</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>LibraryLabel</zorder>
  <zorder>LoginButton</zorder>
  <zorder>Draglabel</zorder>
  <zorder>ToMinButton</zorder>
  <zorder>CloseButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DraggableLabel</class>
   <extends>QLabel</extends>
   <header>cell/draggablelabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>UserEdit</tabstop>
  <tabstop>PasswordEdit</tabstop>
  <tabstop>LoginButton</tabstop>
  <tabstop>ToMinButton</tabstop>
  <tabstop>CloseButton</tabstop>
 </tabstops>
 <resources>
  <include location="../qrc.qrc"/>
 </resources>
 <connections/>
</ui>
